Formula/djl-serving.rb

```diff
@@ -46,15 +46,16 @@ def plist
     assert_predicate testpath/"#{libexec}/lib/serving-0.11.0.jar", :exist?
     http_port = free_port
     puts http_port
-    # system("sh -c 'export DJL_CACHE_DIR=testpath; #{bin}/djl-serving > /dev/null 2>&1 &'")
     fork do
+      system("echo inference_address=http://127.0.0.1:6035 > config.properties")
+      system("echo management_address=http://127.0.0.1:6035 >> config.properties")
+      ENV["SERVING_OPTS"]="-Djna.debug_load=true -Djna.debug_load.jna=true"
       with_env(DJL_CACHE_DIR: testpath) do
```

Comment:
```suggestion
      (testpath/"config.properties").write <<~EOS
        echo inference_address=http://127.0.0.1:6035
        echo management_address=http://127.0.0.1:6035
      EOS
      with_env(
        DJL_CACHE_DIR: testpath,
        SERVING_OPTS: "-Djna.debug_load=true -Djna.debug_load.jna=true"
      ) do
```
You can probably write `config.properties` outside the `fork do` block. Can we use `free_port` instead of `6035`?

Reply:
Yes, I'm working on using the `free_port`. Sure, I'll take the config.properties out of the fork block. We're facing some problem with JNA library where they're not downloading during the build run, whereas it's running on my system.

Reply:
Great. You can do something like
```ruby
port = free_port
(testpath/"config.properties").write <<~EOS
  echo inference_address=http://127.0.0.1:#{port}
  echo management_address=http://127.0.0.1:#{port}
EOS
```

Reply:
Hello, I wanted to ask you about the permissions on the build. For my code to run/pass, I would need write access on the machine. Do we have such access when dealing with tests, `testpath` on the build ?

Reply:
Yes, you have write access to `testpath` and its subdirectories. You almost certainly don't have write access elsewhere.